core.yaml

```diff
@@ -130,6 +178,14 @@ adminApps:
         svc: gitea-http
         port: 3000
         type: public
+    info:
+      appVersion: 1.15.8
+      repo: https://github.com/go-gitea/gitea
+      maintainers: Gitea
+      relatedLinks: [https://otomi.io/docs/apps/gitea ,https://docs.gitea.io/en-us/]
```

Comment:
Please change reformat relatedLinks to much single formatting in YAML document.
Example:
Instead of this:
```
relatedLinks: [https://otomi.io/docs/apps/gitea ,https://docs.gitea.io/en-us/]
```
Do that:
```
relatedLinks: 
- https://otomi.io/docs/apps/gitea
- https://docs.gitea.io/en-us/
```